basics/data-import/pinot-file-system/amazon-s3.md

```diff
@@ -10,7 +15,7 @@ You can enable [Amazon S3](https://aws.amazon.com/s3/) Filesystem backend by inc
 By default Pinot loads all the plugins, so you can just drop this plugin there. Also, if you specify `-Dplugins.include`, you need to put all the plugins you want to use, e.g. `pinot-json`, `pinot-avro` , `pinot-kafka-2.0...`
 {% endhint %}
 
-You can also configure the S3 filesystem using the following options:
+You can configure the S3 filesystem using the following options:
```

Comment:
```suggestion
You can configure the S3 file system using the following options:
```

Reply:
I swear I did a search for all the non-desired variations on "file system". How I missed so many is confusing. Sorry.